src/main/java/frc/robot/SwerveMotor.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot;
+
+public enum SwerveMotor {
+    
+    
+
+    MK4(false, false, false, false, 
+    12.8, 8.142857),
+    MK4I(false, false, false, false, 
```

Comment:
Is it really the case that none of the motors is inverted in either of the models? Can you please check?